<?xml version='1.0'?>
<sdf version="1.4">
  <model name="source">
<pose>-15 -14 0 0 0 0</pose> 
    <static>true</static>
    <link name="link">
      <inertial>
        <mass>1.0</mass>
        <inertia> <!-- inertias are tricky to compute -->
          <!-- http://gazebosim.org/tutorials?tut=inertia&cat=build_robot -->
          <ixx>0.083</ixx>       <!-- for a box: ixx = 0.083 * mass * (y*y + z*z) -->
          <ixy>0.0</ixy>         <!-- for a box: ixy = 0 -->
          <ixz>0.0</ixz>         <!-- for a box: ixz = 0 -->
          <iyy>0.083</iyy>       <!-- for a box: iyy = 0.083 * mass * (x*x + z*z) -->
          <iyz>0.0</iyz>         <!-- for a box: iyz = 0 -->
          <izz>0.083</izz>       <!-- for a box: izz = 0.083 * mass * (x*x + y*y) -->
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <box>
            <size>0.1 0.1 0.1</size>
          </box>
        </geometry>
      </collision>
      <visual name="visual">
        <pose> 0 0 3 0 0 0</pose>
        <geometry>
          <box>
            <size>0.2 0.2 0.2</size>
          </box>
        </geometry>
        <material>
          <ambient>1.0 0 0 1.0</ambient>
					<diffuse>1.0 0 0 1.0</diffuse>
        </material>
      </visual>
      <sensor name="radiation_source" type="radiation_source">
        <always_on> 1 </always_on>
        <update_rate> 1.0 </update_rate>
<topic>source_1/alpha</topic> 
       <plugin name="radiation_source_plugin" filename="libradiation_source_plugin.so"/>      
       
       </plugin>  
      
      </sensor>

    </link>

  </model>

</sdf>
